### DISCUSSION TITLE
Allow using `ctrl + d` to close a buffer from buffer picker by default

### DISCUSSION THREAD

**Discussion by Zaplae** [helix-editor/helix#10430] [Ideas]
`ctrl + d` is equals to `:buffer_close`, this is quick by input `buffer_close`
if you have unsaved buffer, you have to input `:bufer_close!` :)

**Comment by Zaplae** [comment]
Ok, I just did't know it's alias and now the display of aliases was fixed.
I'll close this issue.

**Comment by the-mikedavis** [comment]
Let's keep this open as a `C-d` or similar keymap in the buffer picker could be useful to quickly close a handful of buffers.

**Comment by mangas** [comment]
@sudormrfbin might be worth renaming the ticket as it looks like the purpose has changed?

**Comment by sudormrfbin** [comment]
The title for this issue is referring to closing buffers from the buffer picker using a keybind and I think you opened a ticket with the same intent? If so the title matches since it's still about adding a keybind to the buffer list.

**Comment by mangas** [comment]
Yep you're right, I got a little confused with the alias related discussion above, nevermind!

**Comment by pickfire** [comment]
If anything I think it is better to add shortcuts to `space + b` and have a shortcut key there to delete buffer, rather than having another popup created just to delete buffer.

**Comment by kirawi** [comment]
I added a `handle_key_events` closure field to the `Picker` to have custom keys for each type of picker. It works, but I need to find a way to update the picker afterwards.

**Comment by pickfire** [comment]
I think later on we need some kind of infobox mechanisms to make those keys discoverable.

**Comment by noahfraiture** [comment]
Helli, it would be great. Any news ?

**Comment by Septias** [comment]
Any progress?
